src/Symfony/Component/HttpKernel/CacheClearer/Psr6CacheClearer.php

```diff
@@ -46,6 +46,11 @@ public function clearPool($name)
         return $this->pools[$name]->clear();
     }
 
+    public function getPoolNames()
```

Comment:
Can we add a return type here?

Reply:
I did not add return type here because the other methods didn't have any. But I did consider that. Im not sure what the policy is here. 

Anyhow, this change is outdated.